group_vars/all

```diff
@@ -124,6 +124,17 @@
   matlab_module_appdir: "matlab"
   matlab_module_file: "r2018a"
   matlab_ver: "{{ matlab_module_file }}"
+  matlab_module_name: "rc/matlab"
+  matlab_vers:
+    - 'R2012a'
+    - 'R2013a'
+    - 'R2015a'
+    - 'R2015b'
+    - 'R2016a'
+    - 'R2016b'
+    - 'R2017a'
+    - 'R2017b'
+    - 'R2018a'
```

Comment:
Please set R2018a as the default version in the form dropdown (and it may be better to reverse the order so the newer ones are at the top.